osmtogtfs/gtfs/gtfs_dummy.py

```diff
@@ -123,32 +138,31 @@ def _create_dummy_trip_stoptimes(trip_id, stop_ids, sequence):
         yield {'trip_id': trip_id,
                'arrival_time': '{:02}:{}'.format(arrival_hour, arrival.strftime('%M:%S')),
                'departure_time': '{:02}:{}'.format(departure_hour, departure.strftime('%M:%S')),
-               'stop_id': stop_id,
+               'stop_id': stop.stop_id,
                'stop_sequence': stop_sequence}
 
         stop_sequence += 1
```

Comment:
@mehdisadeghi since you have so many loops in your code that need indexing it might be useful to use the `enumerate` function to avoid manual index handling ;)